sys/include/bitfield.h

```diff
@@ -89,6 +89,69 @@ static inline bool bf_isset(uint8_t field[], size_t idx)
     return (field[idx / 8] & (1u << (7 - (idx % 8))));
 }
 
+/**
+ * @brief  Perform a bitwise OR operation on two bitfields
+ *         `out = a | b`
+ *
+ * @param[out]    out   The resulting bitfield
+ * @param[in]     a     The first bitfield
+ * @param[in]     b     The second bitfield
+ * @param[in]     len   The size of the bitfield in bytes
+ */
```

Comment:
I think we should add a `@pre` stating that the size for a, b, and c must be greater than `len`.

Also, I would rephrase to something like
```
  * @param[in]     len   The number of bytes to which the operation is applied.
```

Reply:
> the size for a, b, and c must be greater than `len`.

Shouldn't it be *at least* `len`? 
I assumed one would typically run the operation on the whole bitfield, not arbitrary byte-sized chunks.

Reply:
Sure, it should just be documented exactly to make sure that it is the responsibility of the caller to check these preconditions.

Reply:
I feel like this is stating the obvious, but [there](https://github.com/RIOT-OS/RIOT/compare/5c0cb8bed44e88f853fcfdd7919a7122020a7a7a..c55cd51e8ef52acc74523dae20bda4b293a39d80) you go